bundler.d/assets.rb

```diff
@@ -16,4 +16,6 @@
   gem 'multi-select-rails', '~> 0.9'
   gem 'gridster-rails', '~> 0.1'
   gem 'jquery_pwstrength_bootstrap', '~> 1.2'
+  gem 'turbolinks', '~> 2.5'
```

Comment:
This has runtime functionality I think?  If so it should be in the main Gemfile and only jquery-turbolinks (which contains only JS that'll be precompiled) should be in assets.